cni/network/plugin/main.go

```diff
@@ -295,15 +294,6 @@ func main() {
 		os.Exit(0)
 	}
 
-	log.SetName(name)
```

Comment:
These were added to support a known issue(incompatilbility of loggers between cns/cni) if i remember correctly ? Is that no longer valid ?

Reply:
yes after we migrate all legacy logs to zap logger, we do not need this anymore and it is using legacy logs, trying to rotate log files but in the meantime, zap logger lumberjack is rotating log files as well, causing windows pods to be crashed and cannot get back. So this PR is to fix this issue.

Reply:
Let's run CNI release pipeline with this change too

Reply:
yes it's running